### ISSUE TITLE
isNaN test of myPayload in I2COutNode can lead to unexpected results

### ISSUE THREAD

**Issue by kpm-at-hfi** [nielsnl68/node-red-contrib-i2c#39]
Sending data out through the i2c out node can have unexpected results.

`myPayload = {"type":"Buffer","data":[23,16]}` works fine and `!isNaN(myPayload)` (from line 160 of 26-i2c.js) is false as expected.

But `myPayload = {"type":"Buffer","data":[51,48]}` fails and `!isNaN(myPayload)` (from line 160 of 26-i2c.js) is true!

This results in the following error being thrown:

```
9 Sep 12:56:32 - [error] [i2c out:97bc9c52.ad8ce8] RangeError [ERR_INVALID_OPT_VALUE]: The value "NaN" is invalid for option "size"
```

**Comment by kpm-at-hfi** [comment]
This is resolved (I believe) in pull request #40

**Comment by papakpmartin** [comment]
I think this is closable now having manually tested v0.8.4
